### ISSUE TITLE
About the warp direction

### ISSUE THREAD

**Issue by zcong17huang** [haofeixu/aanet#42]
Hello @haofeixu . Thank you for the paper and the code.
In your code [https://github.com/haofeixu/aanet/blob/master/nets/warp.py](url)  line 55
`offset = torch.cat((-disp, torch.zeros_like(disp)), dim=1)  # [B, 2, H, W]`
Here you warp the right img to match the left, so the right one should do right shift operation. So I think here should be `disp`, not `-disp`.  Is it right?

**Comment by haofeixu** [comment]
The disparity map is relative to the left image (i.e., left x coordinate - right x coordinate = disp (positive)), so to find the correponding pixel on the right image, we should do `x - disp`.

**Comment by zcong17huang** [comment]
> The disparity map is relative to the left image (i.e., left x coordinate - right x coordinate = disp (positive)), so to find the correponding pixel on the right image, we should do `x - disp`.

Yes, you are right.  I have found my mistake. Thank you very much.
